pkg/metrics/metrics.go

```diff
@@ -563,6 +568,7 @@ func DefaultMetrics() map[string]struct{} {
 		Namespace + "_" + SubsystemAPILimiter + "_rate_limit":                        {},
 		Namespace + "_" + SubsystemAPILimiter + "_adjustment_factor":                 {},
 		Namespace + "_" + SubsystemAPILimiter + "_processed_requests_total":          {},
+		Namespace + "_arping_requests_total":                                         {},
```

Comment:
Should we put this in the subsystem `SubsystemDatapath`? cc @brb

Reply:
No strong opinion on this. The requests originate from the agent, so `SubsystemDatapath` might be misleading. Maybe we could introduce `SubsystemNodeNeigh` if we plan to expose more metrics there?

Reply:
Now that we are using `node-neigh` in git log titles, I guess we could put this in `SubsystemNodeNeigh`